tasks/es/ine.py

```diff
@@ -84,7 +84,7 @@ def columns(self):
         cusec_id = OBSColumn(
             name=u"Secci\xf3n Censal",
             type="Text",
-            tags=[],
+            tags=[sections['spain'], subsections['boundary'],source,license],
```

Comment:
This is causing duplicate entry in the catalog: https://s3.amazonaws.com/data-observatory-catalog/catalogs/canary-islands-fix/spain/boundary.html